lib/utils/classToInvokable.js

```diff
@@ -10,6 +10,12 @@ function classToInvokable(Class) {
   return new Proxy(Class, {
     apply(Target, thisArg, args) {
       return new Target(...args);
+    },
+    construct(Target, args) {
+      return new Target(...args);
+    },
+    get(target, p) {
```

Comment:
Do we need this `get` trap? Isn't default `get` implementation is same?

Reply:
Yea but it also appears to improve performance for some reason 😛